src/components/common/DataTable/slots-columns.tsx

```
@@ -160,41 +162,45 @@ export const createSlotsColumns = (onDeleteSlot?: (slot: Slot) => void): ColumnD
           };
           break;
         default:
+          const statusStr = String(status || 'unknown');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)